department-of-veterans-affairs/next-build
# VACMS-16476: production tag workflow

## Description
Relates to https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16476

## Testing done
This was tested by running the workflow against the Next Build repo to generate tags. There were a number of both unsuccessful and successful runs, all of which can be seen here: https://github.com/department-of-veterans-affairs/next-build/actions/workflows/production-tag.yml

The resulting tags are here: https://github.com/department-of-veterans-affairs/next-build/tags
Every tag after the first one was created by this workflow.

Both success and failure are being reported into this Slack channel: https://dsva.slack.com/archives/C06DSBT7CBW

## Screenshots


## QA steps
There is not a good way to test this without enabling the workflow for Pull Request events; this is how the testing/development was done in the first place. I would recommend reviewing the workflow runs to see how the workflow played out. 

There are also explanatory comments on the file itself.

```[tasklist]
```

## Is this PR blocked by another PR?
- Add the `DO NOT MERGE` label
- Add links to additional PRs here:
